src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -98,30 +98,41 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives with
             }
           }
         } ~
-          pathPrefix("resource") {
-            path(Segment / Segment / Segment / "sync") { (resourceTypeName, resourceId, accessPolicyName) =>
-              val resource = Resource(ResourceTypeName(resourceTypeName), ResourceId(resourceId))
-              val resourceAndPolicyName = ResourceAndPolicyName(resource, AccessPolicyName(accessPolicyName))
-              pathEndOrSingleSlash {
-                post {
-                  complete {
-                    import GoogleModelJsonSupport._
-                    googleExtensions.synchronizeGroupMembers(resourceAndPolicyName).map { syncReport =>
-                      StatusCodes.OK -> syncReport
-                    }
+        pathPrefix("resource") {
+          path(Segment / Segment / Segment / "sync") { (resourceTypeName, resourceId, accessPolicyName) =>
+            val resource = Resource(ResourceTypeName(resourceTypeName), ResourceId(resourceId))
+            val resourceAndPolicyName = ResourceAndPolicyName(resource, AccessPolicyName(accessPolicyName))
+            pathEndOrSingleSlash {
+              post {
+                complete {
+                  import GoogleModelJsonSupport._
+                  googleExtensions.synchronizeGroupMembers(resourceAndPolicyName).map { syncReport =>
+                    StatusCodes.OK -> syncReport
                   }
-                } ~
-                get {
-                  complete {
-                    googleExtensions.getSynchronizedDate(resourceAndPolicyName).map {
-                      case Some(date) => StatusCodes.OK -> Option(GroupSyncResponse(date.toString))
-                      case None => StatusCodes.NoContent -> None
-                    }
+                }
+              } ~
+              get {
+                complete {
+                  googleExtensions.getSynchronizedDate(resourceAndPolicyName).map {
+                    case Some(date) => StatusCodes.OK -> Option(GroupSyncResponse(date.toString))
+                    case None => StatusCodes.NoContent -> None
                   }
                 }
               }
             }
           }
+        } ~
+        path("group" / Segment / "sync") { groupName =>
```

Comment:
I thought we decided not to make this end point but to automatically sync the group